depdive/README.md

```diff
@@ -0,0 +1,54 @@
+# Depdive
```

Comment:
You should look to adding badges to the top of the readme similar to what was done here: https://github.com/bmwill/diffy